docs/api-guide/serializers.md

```diff
@@ -524,6 +524,7 @@ You can add extra fields to a `ModelSerializer` or override the default fields b
 
         class Meta:
             model = Account
+            fields = ['get_absolute_url', 'groups']
```

Comment:
```suggestion
            fields = ['url', 'groups']
```

Reply:
I think it needs to be this?

Reply:
that's what i thought first, but i get an error `Field name ** is not valid for model **`

Reply:
Here is a copy paste of the code i used for the tests: 
```python
# models.py
class Task(models.Model):
    name = models.CharField(max_length=100, unique=True)

    @property
    def get_prop(self):
        return 'eyeyey'
# serializers.py
class TaskSerializer(serializers.ModelSerializer):
    title = serializers.CharField(source='get_prop', read_only=True),

    class Meta:
        model = Task
        fields = ['name', 'get_prop']
```
Tested on django 3.2.9 and DRF 3.12.4

Reply:
But there is still an issue that i haven't seen first, in the result page "title" does not appears, it's "get_prop" instead.
```
[
    {
        "name": "20 pepomm",
        "get_prop": "eyeyey"
    },
]
```

Reply:
So the issue there comes from the fact that `get_prop` is referencing the automatically generated field (thanks to `ModelSerializer`, this would raise an error in a regular `Serializer`). You would need to set it to `url` (or `title` in your example) to get it to use the manually specified field on the serializer instead.

Reply:
Duh, i really don't how i get the error `Field name ** is not valid for model **` the first time, and i was not able to reproduce it. Surely i made a mistake somewhere that confused me, im sorry. Thing is now all seems to be fine and ready to be merged.